### PR TITLE
Spreading additional props on TransitionLink component

### PR DESCRIPTION
**Pull request by aamorozov**

### CODE REVIEW

**Comment by aamorozov** [comment]
@TylerBarnes Any chance you'll have a moment to review sometime this week? Launching a project in a nearest future and would love to use it :)

**Comment by TylerBarnes** [comment]
Hey @aamorozov yes I will for sure! I think I originally had it like this and then it caused issues where anilink was passing things down that shouldn't be passed through and react was complaining. I believe I've already fixed that for an unrelated issue in anilink so this should be good! I'll try it out as soon as I have time this week.
Thanks for this!

File: src/components/TransitionLink.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ const TransitionLink = ({
   activeStyle,
   style,
   className
```
Comment (TylerBarnes) [inline src/components/TransitionLink.js]: Missing a comma here

**Comment by TylerBarnes** [comment]
@aamorozov this looks good! There is a single issue that I mentioned above. Besides that it seems to work fine from my testing. If you can add that comma I'll merge this in.

Thanks again

File: src/components/TransitionLink.js (hunk 1 above)
Comment (aamorozov) [reply src/components/TransitionLink.js]: 🤦‍♂️ That is what happens when you rely on the linters too much... Thank you for catching that!

**Comment by aamorozov** [comment]
@TylerBarnes Thanks for looking at this! I've updated the pr.

**Comment by TylerBarnes** [comment]
Thanks again @aamorozov

**Pull request merged**